src/App.tsx

```diff
@@ -1,7 +1,31 @@
 import { RouterProvider } from 'react-router-dom';
+import { useEffect } from 'react';
 import { router } from 'router';
 import 'App.css';
 
-const App = () => <RouterProvider router={router} />;
+const App = () => {
+  useEffect(() => {
+    const audio = document.querySelector('audio');
+    document.addEventListener(
+      'click',
+      () => {
+        if (audio) {
+          audio.volume = 0.05;
+          audio.play();
+        }
+      },
+      { once: true }
+    );
+  }, []);
```

Comment:
Please remove the listener in `useEffect`'s cleanup
It'll require defining function `
() => {
        if (audio) {
          audio.volume = 0.05;
          audio.play();
        }
      },
      { once: true }
      `
      in `App`'s scope